packages/app/src/app/overmind/actions.ts

```diff
@@ -112,18 +122,18 @@ export const signInButtonClicked: AsyncAction<{
     await actions.internal.signIn({
       useExtraScopes: false,
     });
-    state.signInModalOpen = false;
+    if (state.user) state.signInModalOpen = false;
```

Comment:
I don't think we have to add this check anymore, since we now only return on sign in (if I'm not mistaken?)

Reply:
Good catch! Removed